workshops/pyTelegramWorkshop/README.md

```diff
@@ -0,0 +1,378 @@
+---
+name: 'Connecting Python and Telegram'
+description: 'Connect Telegram with Python to make some awesome stuff'
+author: '@deltaonealpha'
+img: 'https://cloud-cwg1brm1x.vercel.app/'
+---
+
+# Connecting Python and Telegram
+### Hello, world!
+
+# What the result will look like:
+Once done, you will be able to use Telegram BOTs to communicate with you, and your services. It has unlimited opurtunities!
+This is the end product of a small project that this workshop houses:
+
+![screenshot-of-workshop-end-project](https://cloud-fmg4yoxia.vercel.app/0dta29-master.png)
+
+
+Here's a sample of what I personally use this for. My store manager, called DBFA uses a Telegram BOT to inform the store owner about operations, schedules, payments and much more. It also enables two-factor-authentication (2FA) via Telegram.
+![screenshot-of-my-personal-implementation](https://cloud-qnh4qjxv1.vercel.app/0dta30-master.jpg.png)
+
+
+## Hmm... Telegram.. Python?
+
+Yes, Telegram, it is one of the world’s leading messaging apps. Well, “one of the leading apps”. Why Telegram of all?
+
+Well, for starters, it is one of the very few messaging companies that provide open-source API access to everyone. All of Telegram (except the server software) is completely open source!
+
+In fact, the official Telegram app is rather called a client. As its open source, there are literally a gazillion Telegram clients out there.
+
+You don’t like your client? Code one yourself, it is that flexible! 
+
+
+## But Python..?
+
+### Why would one even connect the two?
+
+Python is a leading language today. Being extremely flexible and one of the easiest languages to learn, Python has a lot going for it, and connecting the just makes a lot of sense.
+
+
+
+Let me give you an example.
+
+My father’s friend lost a lot of money because his restaurant’s manager logged meagre sales and pocketed much of the profit.
+
+On hearing this, I started to think, what if every purchase got logged on its own, in a place inaccessible for everyone but the owner? 
+
+Pondering on a way to solve this, I created DBFA, a scalable billing framework written in Python3. It can manage your store, employees, payments, inventory, orders, deliveries, invoicing, and a lot more.
+
+A core feature of DBFA is complete Telegram logging. As soon as a bill is issued, it immediately logs the sales activity in the store owner’s Telegram account as a message from a bot, with all communications over TSL-encrypted HTTPS. Not only this, it even logs every event where store data is being accessed.
+
+
+
+Expanding on this, I even added inline button-based two-factor-authentication as text inputs can be super manipulative.
+
+Yeah, all using Telegram and Python! What I did is nothing compared to what you all can do. Yeah, **you!**
```

Comment:
```suggestion
Let me give you an example.

My father’s friend lost a lot of money because his restaurant’s manager logged meager sales and pocketed much of the profit.

On hearing this, I started to think, what if every purchase got logged on its own, in a place inaccessible for everyone but the owner? 

Pondering on a way to solve this, I created DBFA, a scalable billing framework written in Python3. It can manage your store, employees, payments, inventory, orders, deliveries, invoicing, and a lot more.

A core feature of DBFA is complete Telegram logging. As soon as a bill is issued, it immediately logs the sales activity in the store owner’s Telegram account as a message from a bot, with all communications over TSL-encrypted HTTPS. Not only this, it even logs every event where store data is being accessed.



Expanding on this, I even added inline button-based two-factor-authentication as text inputs can be super manipulative.

Yeah, all using Telegram and Python! What I did is nothing compared to what you all can do. Yeah, **you!**
```